VideoWeb/VideoWeb/ClientApp/src/app/shared/pipes/hyphenate.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+    name: 'hyphenate',
+    pure: false
+})
+export class HyphenatePipe implements PipeTransform {
+    constructor() {}
```

Comment:
remove empty constructor, don't think that will cause an issue?

Reply:
Done